specs/stein/approved/amd-sev-libvirt-support.rst

```diff
@@ -0,0 +1,367 @@
+
+If compression is being used on swap disks then more storage may be
+required because the memory of encrypted VMs will not compress to a
+smaller size.
+
+Other deployer impact
+---------------------
+
+The operator will need to define SEV-enabled flavors as described above
+in order for users to be able to use SEV.
+
+Developer impact
+----------------
+
+None
+
+Upgrade impact
+--------------
+
+None
+
+Implementation
+==============
+
+Assignee(s)
+-----------
+
+Primary assignee:
+  adam.spiers
+
+Other contributors:
+  Various developers from SUSE and AMD
+
+Work Items
+----------
+
+Still TODO
+
+Work items or tasks -- break the feature up into the things that need to be
+done to implement it. Those parts might end up being done by different people,
+but we're mostly trying to understand the timeline for implementation.
+
+
+Dependencies
+============
+
+Still TODO
+
+* Include specific references to specs and/or blueprints in nova, or in other
+  projects, that this one either depends on or is related to.
+
+* If this requires functionality of another project that is not currently used
+  by Nova (such as the glance v2 API when we previously only required v1),
+  document that fact.
+
+* Does this feature require any new library dependencies or code otherwise not
+  included in OpenStack? Or does it depend on a specific version of library?
+
+
+Testing
+=======
+
+Still TODO
+
+Please discuss the important scenarios needed to test here, as well as
+specific edge cases we should be ensuring work correctly. For each
+scenario please specify if this requires specialized hardware, a full
+openstack environment, or can be simulated inside the Nova tree.
+
+Please discuss how the change will be tested. We especially want to know what
+tempest tests will be added. It is assumed that unit test coverage will be
+added so that doesn't need to be mentioned explicitly, but discussion of why
+you think unit tests are sufficient and we don't need to add more tempest
+tests would need to be included.
+
+Is this untestable in gate given current limitations (specific hardware /
+software configurations available)? If so, are there mitigation plans (3rd
+party testing, gate enhancements, etc).
+
+
+Documentation Impact
+====================
+
+Still TODO
+
+Which audiences are affected most by this change, and which documentation
+titles on docs.openstack.org should be updated because of this change? Don't
+repeat details discussed above, but reference them here in the context of
+documentation for multiple audiences. For example, the Operations Guide targets
+cloud operators, and the End User Guide would need to be updated if the change
+offers a new feature available through the CLI or dashboard. If a config option
+changes or is deprecated, note here that the documentation needs to be updated
+to reflect this specification's change.
+
+
+Future work
+===========
+
+Looking further ahead, there is scope for several strands of additional
```

Comment:
explain that htis is outside scope of Stein? we need to give the "scope" of the work for the next cycle so that upstream can understand when the feature is "done" and when all features are merged, which is important for the feature freeze milestone in the roadmap. as such we need to be very explicit on when we consider the "stein implementation level" complete.

Reply:
Yes thanks good point, addressed.  This also applies to the "immediate follow-up work" above regarding the built-in trait.  I will try to touch base with Melanie to get advice on this.